inc/pagination.php

```diff
@@ -32,34 +30,42 @@ function nightingale_archive_pagination() {
 
 	if ( $paginate ) {
 
-		$pagination = '<div class="navigation"><nav class="nhsuk-pagination" role="navigation" aria-label="Pagination"><ul class="nhsuk-list nhsuk-pagination__list">';
+		$pagination = '<hr class="govuk-section-break govuk-section-break--m govuk-section-break--visible"><nav class="moj-pagination" role="navigation" id="pagination-label"><p class="govuk-visually-hidden" aria-labelledby="pagination-label">Pagination navigation</p><ul class="moj-pagination__list">';
 
 		$count = count( $paginate ) - 1;
 
 		$first_item = array_slice( $paginate, 0, 1 )[0];
 		$last_item  = array_slice( $paginate, $count, 1 )[0];
 
-		if ( false !== strpos( $last_item, 'class="next page-numbers"' ) ) {
-
-			$pagination .= "<li class='search__pagination-button search__pagination-button--next'>{$last_item}</li>";
-			array_pop( $paginate );
-		}
+    for ($i=0;$i<count( $paginate );$i++) {
+      if (false !== strpos( $paginate[$i], 'class="prev page-numbers"' )) {
+        $pagination .= "<li class='moj-pagination__item moj-pagination__item--prev'>".str_replace("prev page-numbers", "moj-pagination__link", "{$paginate[$i]}")."</li>";
+      } elseif (false !== strpos( $paginate[$i], 'class="next page-numbers"' )) {
+        $pagination .= "<li class='moj-pagination__item moj-pagination__item--next'>".str_replace("next page-numbers", "moj-pagination__link", "{$paginate[$i]}")."</li>";
+      } elseif (false !== strpos( $paginate[$i], 'class="page-numbers current"' )) {
+        $pagination .= "<li class='moj-pagination__item moj-pagination__item--active'>{$paginate[$i]}</li>";
+      } else {
+        $pagination .= "<li class='moj-pagination__item'>".str_replace("page-numbers", "moj-pagination__link", "{$paginate[$i]}")."</li>";
+      }
+    }
 
 		if ( false !== strpos( $first_item, 'class="prev page-numbers"' ) ) {
-
-			$pagination .= "<li class='search__pagination-button search__pagination-button--previous'>{$first_item}</li>";
 			array_shift( $paginate );
-
+		}
+    if ( false !== strpos( $last_item, 'class="next page-numbers"' ) ) {
+			array_pop( $paginate );
 		}
 
-		$pagination .= "<li class='search__pagination-text'>";
+		$pagination .= '</li></ul>';
 
-		$pagination .= "Page ";
+		$pagination .= ' <p class="moj-pagination__results">';
+
+		$pagination .= "Showing page <b>";
 		$pagination .= (get_query_var('paged')) ? get_query_var('paged') : 1;
-		$pagination .= " of ";
+		$pagination .= "</b> of <b>";
```

Comment:
Is there CSS we could use instead of the `<b>` tag?

Reply:
We don't want to emphasise anything here - so `<strong>` would be wrong.  There is no semantic meaning, so whilst we could use a CSS style, e.g. `<span class="embolden">...</span>`, `<b>` is just a shorthand way of doing that.